### ISSUE TITLE
Failed to fund using the Stellar laboratory Friendbot

### ISSUE THREAD

**Issue by ericchaucl** [stellar/go-horizon#368]
Hi,

I've setup a blank new Stellar network with Horizon up and running. 

I then try to setup the laboratory with IP changed to my internal network. When I create a new account and  click the button "Get test network lumen", it said "Failed to fund GBCBIFNTGODEELYUIBHE4DSAREF3WWWM6SUWSYVBZLSUUDRD2E7CIFB5 on the test network" with below info:

{
  "type": "https://stellar.org/horizon-errors/server_error",
  "title": "Internal Server Error",
  "status": 500,
  "detail": "An error occurred while processing this request.  This is usually due to a bug within the server software.  Trying this request again may succeed if the bug is transient, otherwise please report this issue to the issue tracker at: https://github.com/stellar/horizon/issues. Please include this response in your issue.",
  "instance": "ericvm/8Ffu7YJsyX-000007"
}

I've double check the configuration file, search the web and log but no luck. Appreciate for any further information and help on this. Thanks!

**Comment by bartekn** [comment]
You can only use SDF horizon-testnet to fund accounts in testnet. We do not share secret key of friendbot account.

**Comment by ericchaucl** [comment]
Maybe I do not state clearly, let me state the background more clearly:
(1) I've setup a private Stellar Network
(2) I've changed the source code in src/constants/network.js to point to my private Stellar network
(3) I've changed the friendbot-secret key to my own root account seed when starting up Horizon

We have done this about half an year ago and is working. Just maybe there are some update version of Stellar/Laboratory/Horizon and now doesn't work. 

Or is there any instructions teaching us to host laboratory ourselves to make everything private, instead of using the public/test stellar network?
